src/components/form/importSection.svelte

```diff
@@ -27,18 +36,48 @@
       fr.readAsText(file);
     }
   }
+
+  /**
+    Loads a Json Resume file into MegaResume
+    @param {Event} e fires when uploading a file
+  */
+  function loadJsonResumeFile(e: Event) {
```

Comment:
I'm basically doing the same thing as in the function above, aside from the last part of it. This may be nitpicky, but perhaps I can simplify this into a smaller function.